Resources/Prototypes/_White/Recipes/hidden_crafts.yml

```
@@ -0,0 +1,11 @@
+- type: constructionGraph
+  id: EnergyDoubleSwordGraph
+  start: esword
+  graph:
+  - node: esword
+    edges:
+      - to: desword
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)